services/modules/helm-charts/charts/guardian-auth-service/templates/deployment.yaml

```diff
@@ -44,48 +44,51 @@ spec:
           command: ["/bin/sh"]
           args: ["entrypoint.sh"]
           livenessProbe:
-            {{- toYaml .Values.global.livenessProbe | nindent 12 }}
+            {{- toYaml .Values.livenessProbe | nindent 12 }}
           readinessProbe:
-            {{- toYaml .Values.global.readinessProbe | nindent 12 }}
-          resources:
-            {{- toYaml .Values.resources | nindent 12 }}
+            {{- toYaml .Values.readinessProbe | nindent 12 }}
+          resources: 
+            limits:
+              cpu: "{{ .Values.resources.cpu }}"
+              memory: "{{ .Values.resources.memory }}"
+            requests:
+              cpu: "{{ .Values.resources.cpu }}"
+              memory: "{{ .Values.resources.memory }}"
           envFrom:
             - configMapRef:
                 name: {{ include "guardian-auth-service.fullname" . }}-{{ .Release.Revision }}
             - secretRef:
                 name: {{ include "guardian-auth-service.fullname" . }}-{{ .Release.Revision }}
             - configMapRef:
-                name: {{ .Release.Name }}-tymlez-extensions-newrelic
+                name: guardian-extensions-newrelic
```

Comment:
Shouldnt this be dynamic based on the variable used to choose which APM to enable via $ENABLE_APM_NAME?

How would someone else add a new APM to this?

Reply:
when we add new APM it requires to update heml chart to handle that new APM, 
- new configmap need to be add
- update entrypoint.sh to handle new APM
- everything else need to update for that APM works, such as datadogs we may need to enable agent sidecar